pkg/controller/v1beta1/inferenceservice/controller.go

```diff
@@ -338,6 +342,11 @@ func (r *InferenceServiceReconciler) SetupWithManager(mgr ctrl.Manager, deployCo
 		return err
 	}
 
+	kedaFound, err := utils.IsCrdAvailable(r.ClientConfig, kedav1alpha1.SchemeGroupVersion.String(), constants.KedaScaledObjectKind)
```

Comment:
I am not sure this line is still necessary, since the CRD availability was verified in file `cmd/manager/main.go`.